src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -1902,7 +1906,12 @@ static void _styleInherit(SvgStyleProperty* child, const SvgStyleProperty* paren
         child->stroke.paint.color = parent->stroke.paint.color;
         child->stroke.paint.none = parent->stroke.paint.none;
         child->stroke.paint.curColor = parent->stroke.paint.curColor;
-        child->stroke.paint.url = parent->stroke.paint.url ? _copyId(parent->stroke.paint.url) : nullptr;
```

Comment:
@mgrudzinska 
Existing code sets nullptr when there is no from or parent only in stroke case.
Should I do the same for fill case?

Reply:
I removed the code to set null.
In the test samples, there was no case where the url was set to the child when the parent was null.
Even if there is, there is no need to delete the child's url, and setting nullptr can cause memory leak, so I delete the code.

Reply:
@JSUYA sure! thx